src/main/java/model/Thing.java

```diff
@@ -1,7 +1,16 @@
 package model;
 
 public class Thing {
-    String name;
+    private Long id;
+    private String name;
+    private Long participation_id;
+
+    public Thing(){}
+
+    public Thing(String name, Long participation_id) {
```

Comment:
What is this?

Reply:
Two constructors. One without parameters

Reply:
Python doesn't have it, because it has named parameters

Reply:
Okay, thanks for explanation @mich160